bin/notifications.sh

```diff
@@ -219,6 +219,7 @@ function notification() {
 				log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Created ${JOB_CONTROLE_FILE_BASE}.finished."
 			else
 				log4Bash 'ERROR' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Failed to handle notifications for at least one phase:state combination of ${_project}/${_run}."
+				echo "Failed to handle notifications for at least one phase:state combination of ${_project}/${_run}." >> "${JOB_CONTROLE_FILE_BASE}.started"
```

Comment:
Did zou overbodig en dubbelop moeten zijn. De ```log4bash``` functie controleert of er een ```"${JOB_CONTROLE_FILE_BASE}.started"``` is. Zo ja, dan wordt de log regel ook aan die ```"${JOB_CONTROLE_FILE_BASE}.started"``` toegevoegd. Dat maakt dat je netjes met log levels kunt werken. Een ```echo``` schrijft altijd in die ```"${JOB_CONTROLE_FILE_BASE}.started"```of je nu relatief verbose of juist relatief silent wil loggen.